Library/Formula/cim.rb

```diff
@@ -0,0 +1,652 @@
+class Cim < Formula
+  desc "GNU Simula"
+  homepage "https://www.gnu.org/software/cim/"
+  url "http://ftpmirror.gnu.org/pub/cim/cim-5.1.tar.gz"
```

Comment:
Please add a `https://ftp.gnu.org/gnu/` style mirror as well. You'll find lots of examples in other GNU formulae.